readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Cat0996.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Cat0996.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="954" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/be0de39d-6fa0-40af-bd83-42ed60ca0f60">

Reply:
- [DEMO LINK](https://Cat0996.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Cat0996.github.io/layout_moyo-header/report/html_report/)

Reply:
Hi, you should provide these links not to comments here, but to the PR description.